### PR TITLE
zh-cn: init the translation of Firefix 120 release note

### PR DESCRIPTION
**Pull request by yin1999**
### Description

init the translation of Firefix 120 release note

### CODE REVIEW

File: files/zh-cn/mozilla/firefox/releases/120/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+## 为 Web 开发者带来的变化
+
+### HTML
+
+- 对 `media` 属性的支持被重新引入到 [`<source>`](/zh-CN/docs/Web/HTML/Element/source) 元素，并扩展到包括 `<audio>` 和 `<video>` 元素。该属性最初是在 Firefox 15 中添加的，但在 Firefox 53 中被删除，当时它的使用仅限于 `<picture>` 中的 `<source>` 元素。随着这个版本的发布，`media` 属性将在 `<audio>`、`<video>` 和 `<picture>` 中的 `<source>` 元素中可用（[Firefox bug 1836128](https://bugzil.la/1836128)）。
+
+### CSS
+
+- 现已支持 {{CSSXref("color_value/light-dark", "light-dark()")}} CSS 颜色函数。这允许在不需要 `prefers-color-scheme` 媒体特性的情况下设置浅色和深色的颜色（[Firefox bug 1856999](https://bugzil.la/1856999)）。
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/releases/120/index.md:20]:
```suggestion
- 现已支持 {{CSSXref("color_value/light-dark", "light-dark()")}} CSS 颜色值函数。这允许在不需要 `prefers-color-scheme` 媒体特性的情况下设置浅色和深色的颜色（[Firefox bug 1856999](https://bugzil.la/1856999)）。
```

File: files/zh-cn/mozilla/firefox/releases/120/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+- Firefox 支持 [Global Privacy Control](https://globalprivacycontrol.org/) {{HTTPHeader("Sec-GPC")}} 请求标头，该请求标头可用于指示用户不同意网站或服务将其个人信息出售或与第三方共享。用户可以通过将首选项 `privacy.globalprivacycontrol.enabled` 设置为 `true`（在 `about:config` 中）以在正常浏览和隐私浏览模式下启用该标头。{{domxref("Navigator.globalPrivacyControl")}} 和 {{domxref("WorkerNavigator.globalPrivacyControl")}} 属性允许 JavaScript 检查用户同意首选项（[Firefox bug 1856029](https://bugzil.la/1856029)）。
+
+### 安全
+
+无显著变化
+
+### API
+
+- 现已支持 {{domxref("PublicKeyCredential")}} 接口的 {{domxref("PublicKeyCredential.authenticatorAttachment", "authenticatorAttachment")}} 属性。这允许 Web 应用程序的客户端和服务器代码根据认证器是设备的一部分还是可以在设备之间漫游来配置自身（参见 [Firefox bug 1810851](https://bugzil.la/1810851)）。
+- 已支持 [Web Authentication API](/zh-CN/docs/Web/API/Web_Authentication_API) 的 [最短 PIN 码长度扩展（`minPinLength`）](/zh-CN/docs/Web/API/Web_Authentication_API/WebAuthn_extensions#minpinlength)，这允许依赖方服务器在创建/注册期间请求认证器的最短 PIN 码长度（[Firefox bug 1844450](https://bugzil.la/1844450)）。
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/releases/120/index.md:67]:
```suggestion
- 已支持 [Web Authentication API](/zh-CN/docs/Web/API/Web_Authentication_API) 的[最短 PIN 码长度扩展（`minPinLength`）](/zh-CN/docs/Web/API/Web_Authentication_API/WebAuthn_extensions#minpinlength)，这允许依赖方服务器在创建/注册期间请求认证器的最短 PIN 码长度（[Firefox bug 1844450](https://bugzil.la/1844450)）。
```

File: files/zh-cn/mozilla/firefox/releases/120/index.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+{{FirefoxSidebar}}
+
+本文提供了有关 Firefox 120 中影响开发者的变更信息。Firefox 120 于 [2023 年 11 月 21 日](https://whattrainisitnow.com/release/?version=120)发布。
+
+## 为 Web 开发者带来的变化
+
+### HTML
+
+- 对 `media` 属性的支持被重新引入到 [`<source>`](/zh-CN/docs/Web/HTML/Element/source) 元素，并扩展到包括 `<audio>` 和 `<video>` 元素。该属性最初是在 Firefox 15 中添加的，但在 Firefox 53 中被删除，当时它的使用仅限于 `<picture>` 中的 `<source>` 元素。随着这个版本的发布，`media` 属性将在 `<audio>`、`<video>` 和 `<picture>` 中的 `<source>` 元素中可用（[Firefox bug 1836128](https://bugzil.la/1836128)）。
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/releases/120/index.md:16]:
```suggestion
- 对 `media` 属性的支持被重新引入到 [`<source>`](/zh-CN/docs/Web/HTML/Element/source) 元素，并扩展到包括 `<audio>` 和 `<video>` 元素。该属性最初是在 Firefox 15 中添加的，但在 Firefox 53 中被删除，当时它的使用仅限于 `<picture>` 的 `<source>` 子元素。随着这个版本的发布，`media` 属性将在 `<audio>`、`<video>` 和 `<picture>` 的 `<source>` 子元素上可用（[Firefox bug 1836128](https://bugzil.la/1836128)）。
```

**Review by jasonren0403 — APPROVED** [review]
Thanks @yin1999 👍 
Glad to see the release note come back :(

**Pull request merged**